src/components/App.js

```diff
@@ -19,11 +20,20 @@ let mapStateToProps = (state) => {
 class App extends React.Component {
   constructor(props) {
     super(props)
+    this.state = {
+      isBoardVisible : true,
+      isRulesVisible : false
+    }
   }
 
   socket = io(env.API_BASE_URL + ":" + env.SOCKET_PORT)
+  onSubmit = () => {
```

Comment:
rename 'onSubmit' to something more specific, like, toggleGameRules etc, there might be more buttons in the future.